doc/nrf/releases_and_maturity/migration/migration_guide_2.5.rst

```diff
@@ -25,12 +25,15 @@ The following changes are mandatory to make your application work in the same wa
   * The operations to update bootloader (``3``) and read (``6``) or erase (``8``) the MCUboot secondary slot have been removed from the ``#XFOTA`` AT command.
   * The ``#XSLMUART`` AT command has been removed.
     UART is now configured using only devicetree.
-  * The :kconfig:option:`CONFIG_SLM_CARRIER_APP_DATA_CONTAINER_BUFFER_LEN` Kconfig option has been renamed to :kconfig:option:`CONFIG_SLM_CARRIER_APP_DATA_BUFFER_LEN`.
-  * The ``#XDFUGET``, ``#XDFUSIZE`` and ``#XDFURUN`` AT commands have been removed.
+    UART settings that were previously saved for this command now provoke error logs on startup.
+    They are innocuous.
+    This will be fixed in the next |NCS| release.
```

Comment:
```suggestion

    UART settings that were previously saved for this command now provoke error logs on startup.
    They are innocuous.
    This will be fixed in the next |NCS| release.

```
Adding a little spacing by turning this into its own paragraph under the bullet. Makes it more noticeable in the built docs.